requirements/edx/base.txt

```diff
@@ -35,9 +35,10 @@ git+https://github.com/edx/django-rest-framework.git@3c72cb5ee5baebc432894737119
 django==1.8.9
 djangorestframework-jwt==1.7.2
 edx-django-oauth2-provider==0.5.0
+edx-djangorestframework-oauth==1.0.1
```

Comment:
I would also like to get rid of this fork. Shadowing `django-oauth2-provider` by installing `edx-django-oauth2-provider` after `djangorestframework-oauth` would resolve the issue, but it seems brittle to me. Would a comment about the install order suffice? I haven't found a clean, explicit way to have pip override the dependency.

Changes to the `edx-djangorestframework-oauth` fork were introduced in https://github.com/edx/django-rest-framework-oauth/pull/1 during the DRF upgrade. @macdiesel any chance you have something to add here?